README.md

```diff
@@ -2,13 +2,13 @@
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/260964318a4e4e689cbd6d059472765e)](https://www.codacy.com/app/davidwallen/bundle-linter?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=apigeecs/bundle-linter&amp;utm_campaign=Badge_Grade)
```

Comment:
fy: Codacy is report is stale. Actual results are a C score now. There is a bunch of es linter warnings on the project now.

Reply:
Thanks, will take a look